pxr/base/arch/systemInfo.cpp

```diff
@@ -89,10 +89,7 @@ _DynamicSizedRead(
     const std::function<bool(char*, size_t*)>& callback)
 {
     // Make a buffer for the data.
-    // We use an explicit deleter to work around libc++ bug.
```

Comment:
It appears the fix associated with the linked bug has not yet been released?

https://github.com/llvm/llvm-project/commit/85d3e7a7299f0db3286cf0b611bdfac79f39e967

Reply:
My bad, thought closed bug means fix is merged.

Dropped my commit, thanks!

Reply:
It's hard to track LLVM bugs because they transitioned to a new system, and that bug is from the old system. As best as I can tell the fix is in a preview release of an upcoming llvm, but that is, as best as I can tell ;)